host/capture_control.cpp

```
@@ -0,0 +1,245 @@
+    return "FAILED";
+}
+
+}  // namespace
+
+CaptureControlEndpoint::CaptureControlEndpoint(
+    supervisor::IWorkerSupervisor& supervisor)
+    : supervisor_(supervisor)
+{
+}
+
+void CaptureControlEndpoint::RegisterLaunchSpec(
```

Cppcheck (2.21.0)

[style] The function 'RegisterLaunchSpec' is never used.

(unusedFunction)